dali/test/python/test_optical_flow.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import print_function
+from __future__ import division
+import os
+import numpy as np
+import shutil
+import sys
+from nvidia.dali.pipeline import Pipeline
+import nvidia.dali.ops as ops
+import nvidia.dali.types as types
+from random import shuffle
+
+batch_size = 1
+sequence_length = 10
+dali_extra_path = os.environ['DALI_EXTRA_PATH']
+image_dir=dali_extra_path + "/db/optical_flow/slow_preset/"
+data_dir=image_dir + "/data"
+if not os.path.exists(data_dir):
+    os.makedirs(data_dir)
+    os.makedirs(data_dir+"/0")
+    
+
+def copy_image(source,target):
+    try:
+       shutil.copy(source, target)
+    except IOError as e:
+       print("Unable to copy file. %s" % e)
+    except:
+       print("Unexpected error:", sys.exc_info())
+       
+copy_image(image_dir + "/frame_input.png", data_dir+"/0/0001.png")
+copy_image(image_dir + "/frame_reference.png", data_dir+"/0/0002.png")
```

Comment:
You can modify DALI_extra to have:
`/db/optical_flow/two_frames/0/0001.png`, `optical_flow/two_frames/0/0002.png`
and skip all this mkdir and file copy.

Than use just `data_dir=dali_extra_path + "/db/optical_flow/two_frames"`.

Reply:
do you mean i modify DALI_extra directly?

Reply:
https://github.com/NVIDIA/DALI_extra/pull/13